ccd-definition/ComplexTypes/DQ/Hearing.json

```diff
@@ -0,0 +1,45 @@
+[
+  {
+    "ID": "Hearing",
+    "ListElementCode": "hearingLength",
+    "FieldType": "FixedRadioList",
+    "FieldTypeParameter": "HearingLength",
+    "ElementLabel": "Do you want to use an expert?",
+    "SecurityClassification": "Public"
+  },
+  {
+    "ID": "Hearing",
+    "ListElementCode": "hearingLengthHours",
+    "FieldType": "Number",
+    "ElementLabel": "How many hours?",
+    "HintText": "Enter a numeric value, for example 3",
```

Comment:
is there validation on this field? i.e must be less than 6 or something?

Reply:
Not part of the AC so no